Resources/Prototypes/Entities/Objects/Weapons/Guns/Battery/battery_guns.yml

```
@@ -519,6 +519,53 @@
+  parent: BaseWeaponBatterySmall
+  id: WeaponCivilianDisabler
+  description: The original prototype disablers refitted for civilian self defence use and put in a shiny new shell. It has a much lower power output than the modern disabler in use by Security, and as such requires no license to carry.
+  components:
+    - type: Tag
+      tags:
+        - Taser
+        - Sidearm
+    - type: Item
+      size: Small
+      shape:
+      - 0,0,1,1
```

YAMLlint (1.35.1)

[error] wrong indentation: expected 8 but found 6

(indentation)